components/forms/PasswordResetForm.js

```diff
@@ -0,0 +1,52 @@
+import { useRef } from 'react';
+import { Form, FormInput, FormButtons, PasswordField, Button } from 'react-basics';
+import { useApi } from 'next-basics';
+import { useMutation } from '@tanstack/react-query';
+import { getAuthToken } from 'lib/client';
+import styles from './Form.module.css';
+
+export default function PasswordResetForm({ token, onSave }) {
```

Comment:
I don't think this needs to be here. There is no email functionality in analytics.